odl/set/domain.py

```diff
@@ -224,7 +224,8 @@ def __eq__(self, other):
 
     def __hash__(self):
         """Return ``hash(self)``."""
-        return hash(type(self)) ^ hash(self.min_pt) ^ hash(self.max_pt)
+        return (hash(type(self)) ^
+                hash(list(self.min_pt)) ^ hash(list(self.max_pt)))
```

Comment:
Build system noticed that as well, went with tuple, but perhaps should do tostring. Felt tuple is more readable however.